pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -158,22 +161,25 @@ public class ServiceConfiguration implements PulsarConfiguration {
     @FieldContext(
             category = CATEGORY_SERVER,
             required = false,
-            doc = "The metadata store URL for the configuration data. If empty, we fall back to use metadataStoreUrl"
+            doc = """
+                    The metadata store URL for the configuration data. If empty, we fall back to use metadataStoreUrl"""
     )
     private String configurationMetadataStoreUrl;
 
     @FieldContext(
         category = CATEGORY_SERVER,
-        doc = "The port for serving binary protobuf requests."
-            + " If set, defines a server binding for bindAddress:brokerServicePort."
-            + " The Default value is 6650."
+        doc = """
+             The port for serving binary protobuf requests. \
+             If set, defines a server binding for bindAddress:brokerServicePort. \
+             The Default value is 6650."""
```

Comment:
If we need a `\` to escape line break here, I don't think it's a good fit to use text block. Ditto others.

For those who need a pre-formatted doc string, text blocks can be nice.

Reply:
I think it is more readable than strings joined, maybe we can by adding a line break after `.` to reduce the use of ` \ `.

Reply:
Also can fully left justify a wide string in order to avoid horizontal scrolling or line wrapping, see: https://docs.oracle.com/en/java/javase/15/text-blocks/index.html#style-guidelines-for-text-blocks  G10

Reply:
The original string context is not "strings joined", it's "strings concatenation". So, please leave the content as is.

If you'd like to add a newline by using textblock. I don't have a strong feeling here. Do as you like.

To sum up, either:

1. Revert the change;
2. Remove `\`.

is OK for me. `\` doesn't work because we do a simple thing more complex.

Reply:
Ok, I will only leave which multi-line string and include escape character changes.